components/shared/Header.js

```diff
@@ -0,0 +1,156 @@
+import React from 'react';
+
+export default function Header(props) {
+  return (
+    <>
+      <nav className={'navbar navbar-default navbar-trans navbar-expand-lg fixed-top ' + props.theme}>
+        <div className="container">
+          <button
+            aria-controls="navbarDefault"
+            aria-expanded="false"
+            aria-label="Toggle navigation"
+            className="navbar-toggler collapsed"
+            data-bs-target="#navbarDefault"
+            data-bs-toggle="collapse"
+            type="button"
+          >
+            <span />
+            <span />
+            <span />
+          </button>
+          <img
+            alt="image"
+            className="d-none d-lg-block image"
+            height="60"
+            src={'/img/meb-mini/' + props.projectName + '.svg'}
+            width="180"
+          />
+          <div className="navbar-collapse collapse justify-content-center" id="navbarDefault">
+            <ul className="navbar-nav">
+              <li className="nav-item dropdown">
+                <a
+                  aria-expanded="false"
+                  aria-haspopup="true"
+                  className="nav-link dropdown-toggle"
+                  data-bs-toggle="dropdown"
+                  href="#"
+                  id="navbarDropdown"
+                  role="button"
+                >
+                  English
+                </a>
+                <div className="dropdown-menu">
+                  <a className="dropdown-item ">
+                    Deutsch
+                  </a>
+                  <a className="dropdown-item ">
+                    English
+                  </a>
+                  <a className="dropdown-item ">
+                    Français
+                  </a>
+                  <a className="dropdown-item ">
+                    Italiano
+                  </a>
+                  <a className="dropdown-item ">
+                    Nederlands
```

Comment:
The list of languages here are dynamic (configured in DBDefs.pm for MB) and wouldn't apply to projects outside of MB.  Given all the translatable strings in this file and project-specific implementation details, I don't see how we can have a shared component this specific -- it'd have to be much more generalized.